presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -168,6 +168,10 @@ Unit              Description
 
     Returns ``timestamp2 - timestamp1`` expressed in terms of ``unit``.
 
+.. function:: date_diff(unit, unixtime1, unixtime2) -> bigint
+
+    Returns ``unixtime2 - unixtime1`` expressed in terms of ``unit`` (unsupported units: [``millisecond``, ``month``, ``quarter``, ``year``]).
```

Comment:
```suggestion
    Returns ``unixtime2 - unixtime1`` expressed in terms of ``unit``. 
    
    Unsupported units for this function are  ``millisecond``, ``month``, ``quarter``, and ``year``.
```
Suggest separating parenthesis into a separate sentence for easier readability, and a line break for the second sentence to emphasize the limitation. Here's a screenshot of this and the preceding function in a local build showing what my suggestion results in: 
<img width="913" alt="Screenshot 2024-03-13 at 1 18 16 PM" src="https://github.com/prestodb/presto/assets/7013443/a7ea8667-b2ca-4029-a494-2a61487b10a1">

Reply:
@steveburnett LGTM, I will push the update soon. Thanks for your suggestion!